### ISSUE TITLE
Vincent 2016 metadata does not reflect updated metadata

### ISSUE THREAD

**Issue by cmirzayi** [waldronlab/curatedMetagenomicData#284]
**Describe the bug**
Vincent 2016's metadata was incorrect as described in the now closed issue on curatedMetagenomicDataCuration: https://github.com/waldronlab/curatedMetagenomicDataCuration/issues/68. The metadata was updated based on a spreadsheet of metadata received from the author; however, these changes are not reflected in curatedMetagenomicData

**To Reproduce**
```
meta <- curatedMetagenomicData::sampleMetadata
submeta <- meta %>% filter(study_name=="VincentC_2016")
submeta$antibiotics_family %>% table()
```

This table does not match what was published in the original paper or in the updated metadata as described in issue 68.

**Expected behavior**
The Vincent 2016 metadata should return the updated metadata and not the original, erroneous metadata.

**Comment by schifferl** [comment]
Thank you @cmirzayi, I will pull in the updated metadata from curatedMetagenomicDataCuration.

**Comment by schifferl** [comment]
Hi @cmirzayi, this issue was resolved by c6e05f4 in the master branch and 0f485b2 / e7c842f in the RELEASE_3_16 branch. It will be a few days before Bioconductor rebuilds the package, but the changes are available now via GitHub.
